libs/digger_config/yaml.go

```diff
@@ -11,6 +11,7 @@ type DiggerConfigYaml struct {
 	Projects                []*ProjectYaml               `yaml:"projects"`
 	AutoMerge               *bool                        `yaml:"auto_merge"`
 	Workflows               map[string]*WorkflowYaml     `yaml:"workflows"`
+	Telemetry               *bool                        `yaml:"telemetry,omitempty"`
```

Comment:
Please keep in mind that people who have not read the changelog or the thread about the issues around telemetry might not see this and update their config flag. Since the feature is opt-out, having the old flag set will re-enable telemetry for these users.